### PR TITLE
feat: fragmentFile property for custom extensions

### PR DESCRIPTION
**Pull request by broksy**
Added optional `fragmentName` property to use as keys in manifest templates.

### CODE REVIEW

File: .changeset/angry-roses-draw.md [hunk 1]
```diff
@@ -0,0 +1,5 @@
+---
+'@sap-ux/fe-fpm-writer': patch
+---
+
+Added `fragmentName` property for custom elements type to use it as key if defined in manifest templates
```
Comment (815are) [inline .changeset/angry-roses-draw.md:5]: changelog mentions property `fragmentName`, but in source code it is `fragmentFile`

**Review by 815are — COMMENTED** [review]
I noticed following difference, if we pass  configuration like:
```
{
    "target": "sample",
    "name": "NewCustomSection",
    "folder": "extensions/custom",
    "title": "New Custom Section",
    "position": {
        "placement": "After",
        "anchor": "DummyFacet"
    },
    "fragmentFile": "NewCustomSectionFragment",
    "eventHandler": true
}
```

n result we get following files:
1. fragment -> `NewCustomSectionFragment.fragment.xml` - is OK
2. handler file - `NewCustomSection.js`

problem - created hander file differs from fragment name. Usually we create same names:
`Test.fragment.xml`, 'Test.js'

**Review by 815are — APPROVED** [review]
Checked:
1. changes in code and template looks correct and backward compatible
2. old tests are not affect and new tests are added
3. code affects extensions with fragments - columns, sections, header sections, sub sections. Filter field already supports custom name for fragment
4. changelog presented

**Review by zdravko-georgiev — APPROVED** [review]
Code looks good
Excellent test coverage
Did Not tested manually

**Pull request merged**